webapp-django/crashstats/settings/base.py

```diff
@@ -588,21 +588,8 @@ def filter(self, record):
             "access_key": config("resource.boto.access_key", None),
             "bucket_name": config("resource.boto.bucket_name", "crashstats"),
             "region": config("resource.boto.region", "us-west-2"),
-            "prefix": config("resource.boto.prefix", ""),
-            # NOTE(willkg): In the local dev environment, we need to use a
-            # HostPortS3ConnectionContext which requires these additional configuration bits. The
-            # defaults are taken from the config sections of the relevant classes.
-            "resource_class": config(
-                "resource.boto.resource_class",
-                "socorro.external.boto.connection_context.RegionalS3ConnectionContext",
-            ),
-            "host": config("resource.boto.host", None),
-            "port": config("resource.boto.port", None),
-            "secure": config("resource.boto.secure", True),
```

Comment:
There's a `resource.boto.secure=false` that  can probably be removed from `docker/config/never_on_a_server.env` now

Reply:
Yes! I'll do this.